oml/metrics/embeddings.py

```diff
@@ -276,6 +208,9 @@ def compute_metrics(self) -> TMetricsDict_ByLabels:  # type: ignore
 
         return metric_to_return  # type: ignore
 
+    def ready_to_visualize(self) -> bool:
+        return isinstance(self.dataset, IVisualizableDataset)
```

Comment:
suppose metrics also should be calculated. At least `def visualize(self)` uses metrics